### PR TITLE
Add some news posts

### PR DESCRIPTION
**Pull request by hexylena**
- how easy it is to make a video for @bgruening 
- we left twitter, follow us on mastodon (outdated)
- updated quoting features of the GTN to support citing URLs + author names, it's a standards compliant use of blockquote that basically no one uses, and no client renders, so we patch it in in JS. (sub-optimal for non-web clients, but I don't really feel like doing it in a plugin today.)

image

### CODE REVIEW

File: news/_posts/2023-11-21-video.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (bgruening) [inline news/_posts/2023-11-21-video.md:1]: Lovely! Thanks!

File: news/_posts/2023-11-21-mastodon.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (bgruening) [inline news/_posts/2023-11-21-mastodon.md:1]: Top!

File: topics/visualisation/tutorials/circos/tutorial.md [hunk 3]
```diff
@@ -402,7 +402,7 @@ SVs are usually described in terms of the SV **breakpoints** (or **junctions**);
 ```
 
 
-**Circos Input Format:**
+### Circos Input Format
```
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/visualisation/tutorials/circos/tutorial.md:405]: This is expected, the heading parse doesn't know how to deal with code blocks.

**Comment by hexylena** [comment]
Feel free to merge it @bgruening

**Review by shiltemann — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks! Super cool

**Pull request merged**